apps/analysis/mutation.py

```diff
@@ -269,6 +278,45 @@ class Arguments:
     result = graphene.Field(AnalysisReportUploadType)
 
 
+class CreateAnalysis(RequiredPermissionMixin, PsGrapheneMutation):
+    class Arguments:
+        data = AnalysisInputType(required=True)
+    model = Analysis
+    serializer_class = AnalysisGqlSerializer
+    result = graphene.Field(AnalysisType)
+
+
+class UpdateAnalysis(RequiredPermissionMixin, PsGrapheneMutation):
+    class Arguments:
+        data = AnalysisInputType(required=True)
+        id = graphene.ID(required=True)
+    model = Analysis
+    serializer_class = AnalysisGqlSerializer
+    result = graphene.Field(AnalysisType)
+
+
+class DeleteAnalysis(RequiredPermissionMixin, PsDeleteMutation):
+    class Arguments:
+        id = graphene.ID(required=True)
+    model = Analysis
+    result = graphene.Field(AnalysisType)
+
+
+class BulkAnalysisInputType(AnalysisInputType):
+    id = graphene.ID()
+
+
+class BulkAnalysis(RequiredPermissionMixin, PsBulkGrapheneMutation):
```

Comment:
Maintain the naming of class Name 
 ```suggestion
class BulkUpdarteAnalysis(RequiredPermissionMixin, PsBulkGrapheneMutation):
```